### PR TITLE
Additional notification convenience method

### PR DESCRIPTION
**Pull request by rmigneco**
Adds another convenience method for posting notifications.

### CODE REVIEW

File: Pod/Classes/NotificationConvenience.swift [hunk 1]
```diff
@@ -17,6 +17,11 @@ public extension NotificationCenter {
         }
     }
     
+    public func post(notificationName: Notification.Name, queue: DispatchQueue = DispatchQueue.main, object: Any? = nil, userInfo: [AnyHashable: Any]? = nil) -> Void {
```
Comment (jgrandelli) [inline Pod/Classes/NotificationConvenience.swift]: nice!

Comment (codecaffeine) [inline Pod/Classes/NotificationConvenience.swift]: you should have your `public func post(notificationName: String` call this method

Comment (jgrandelli) [inline Pod/Classes/NotificationConvenience.swift]: 👍

Comment (rmigneco) [inline Pod/Classes/NotificationConvenience.swift]: Good call, updated

**Pull request merged**